manuscript/manuscript.Rmd

```diff
@@ -508,13 +473,12 @@ Rarely do we know the underlying functional forms for ecological processes -- is
 Methods which must first assume something about the structure or functional form of a process before estimating the corresponding parameter can only ever be as good as those structural assumptions.
 Frequently available ecological data is insufficient to distinguish between possible alternative models [@devalpine], or the correct model may be non-identifiable with any amount of data.
 Model-free RL approaches offer a powerful way to factor this thorny issue.
-The model-free RL agent needs to make no such a priori hypothesis about a particular functional form. <!-- ML: I feel like the remaining part of this dont full drive home the benefit of model-free rl, like its able to learn good performing policies even in the case when a model is very difficult or impossible to learn. -->
-Alternative models can easily be implemented in simulations, allowing an agent to train against both contingencies.
-A successful agent will learn a policy that works well in both scenarios.
-This is analogous to classical approaches for integrating over parameter or model uncertainty [e.g. @Walters1981; @Ludwig1982],
+The model-free RL agent needs to make no such a priori hypothesis about a particular functional form.
+Model-free approaches have proven successful at learning effective policies even when the underlying model may be difficult or impossible to learn [@alphago], as long as simulations of possible mechanisms are available.
```

Comment:
In an edit that may be lost now on thelio 😬 , I tried to mention here that the concern with the model-free approach is sample complexity -- not a concern if you have a simulator but could be problematic if you are trying to learn directly from data via offline algorithms. 

AlphaGo is also probably not the best citation here -- some aspects I'd call model-based. [https://arxiv.org/pdf/1802.09081.pdf](https://arxiv.org/pdf/1802.09081.pdf) seems to at least hammer the comparison here although does not show it or give a solid citation for it.